<ui version="4.0" >
 <class>ServerGroupDialogUI</class>
 <widget class="QWidget" name="ServerGroupDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>493</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QLabel" name="m_networkLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Network name:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="m_nameEdit" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="m_identityLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Identity:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="m_identityCBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="KPushButton" name="m_editIdentityButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Edit...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="m_commandsLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Commands:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLineEdit" name="m_commandEdit" />
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QCheckBox" name="m_autoConnectCBox" >
     <property name="text" >
      <string>Co&amp;nnect on application start up</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Servers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="6" >
       <widget class="KListWidget" name="m_serverLBox" />
      </item>
      <item row="1" column="0" >
       <widget class="KPushButton" name="m_addServerButton" >
        <property name="text" >
         <string>&amp;Add...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KPushButton" name="m_changeServerButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>E&amp;dit...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QToolButton" name="m_removeServerButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <spacer name="serverSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>228</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4" >
       <widget class="QToolButton" name="m_upServerBtn" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="autoRepeat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QToolButton" name="m_downServerBtn" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="autoRepeat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Auto Join Channels</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" colspan="6" >
       <widget class="KListWidget" name="m_channelLBox" />
      </item>
      <item row="1" column="0" >
       <widget class="KPushButton" name="m_addChannelButton" >
        <property name="text" >
         <string>Add...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KPushButton" name="m_changeChannelButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Ed&amp;it...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QToolButton" name="m_removeChannelButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <spacer name="channelSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>255</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4" >
       <widget class="QToolButton" name="m_upChannelBtn" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="autoRepeat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QToolButton" name="m_downChannelBtn" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="autoRepeat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
